ansible_collections/arista/cvp/docs/release-notes/v3.x.md

```diff
@@ -8,6 +8,26 @@
 - On premise version higher than 2018.2.5
 - Cloudvision as a Service
 
+## Release 3.4.0
+
+Full release note available on [github](https://github.com/aristanetworks/ansible-cvp/releases/tag/v3.4.0)
+
+### Enhancements
+
+- Implement initial module for tags management: `arista.cvp.cv_tag_v3`
+- Implement initial module for changecontrol management: `arista.cvp.cv_change_control_v3`
+- Add support for new auth method for on-perm devices
+
+### Bug fixes
+
+- `cv_image_v3` Warn if image already exists (#471)
+- `cv_container_v3` Fix partial topology removal and container removal
+
+### Deprecation
+
+- Version 01 of `arista.cvp` module are now deprecated and will be removed from next release
```

Comment:
```suggestion
- Version 1 of `arista.cvp` module are now deprecated and will be removed from next release
```

Reply:
Though we already have this sentence in 3.3.0 - what's the status here? Have the files been deleted already? Or are we postponing this for next release?